src/features/landing/components/AboutUs/Info/Info.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+
+import { Stack, Button, Typography } from '@mui/material';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+function Info() {
+  const { t } = useTranslation();
+  return (
+    <Stack maxWidth="701px">
+      <Typography
+        variant="h1"
+        sx={{
+          color: '#1A1C1E',
+          fontFamily: 'Golos',
```

Comment:
its duplicate, you can create a custom typography component with this styles and you will don't need to write a lot of duplicated styles, rework this

Reply:
I'm working on it